orchagent/portsorch.cpp

```diff
@@ -1168,36 +1173,55 @@ bool PortsOrch::setPortAutoNeg(sai_object_id_t id, int an)
     return true;
 }
 
-bool PortsOrch::setHostIntfsOperStatus(const Port& port, bool isUp) const
+bool PortsOrch::setHostIntfsOperStatus(sai_object_id_t port_id, bool up)
```

Comment:
Why changing  from port to port_id ?    It was changed to port for efficiency.

Reply:
this is a new change， and I'll solve the conflict

Reply:
This is a new change， I'll solve the conflict